thirdparty/cmdconfig/commands/cmdeval/cmdeval_test.go

```diff
@@ -462,5 +466,97 @@ func TestCmd_flagAndArgParsing_Symlink(t *testing.T) {
 	assert.Equal(t, filepath.Join("path", "to", "pkg", "dir"), r.RunFns.Path)
 }
 
+func TestAddImageToKptfile(t *testing.T) {
```

Comment:
Pl. take a look at `e2e` tests for `fn eval`. They are under `e2e/testdata/fn-eval`. Add tests for at least the following:

1. On successful `fn eval`, function is added to the pipeline
2. On failure, function is not added
3. --image and --exec 
4. ConfigMap and configPath option
5. selectors if we add support for them